apiv2/db_import/importers/frame.py

```diff
@@ -0,0 +1,53 @@
+import os
+from typing import Any
+
+from database import models
+from db_import.common.finders import FileFinder
+from db_import.importers.base import IntegratedDBImporter, ItemDBImporter
+
+# Frame Fields
+#   deposition_id: int
+#   run_id: int
+#   raw_angle: int
+#   acquisition_order: int
+#   dose: float
+#   is_gain_corrected: bool
+#   s3_frame_path: str
+#   https_frame_path: str
+
+class FrameItem(ItemDBImporter):
+    direct_mapped_fields = {}
+    id_fields = ["run_id", "s3_frame_path"]
+    model_class = models.Frame
+
+    def load_computed_fields(self):
+        https_prefix = self.config.https_prefix
+        s3_prefix = self.config.s3_prefix
+
+        self.model_args["s3_frame_path"] = os.path.join(s3_prefix, self.input_data["file"])
+        self.model_args["https_frame_path"] = os.path.join(https_prefix, self.input_data["file"])
+        self.model_args["run_id"] = self.input_data["run"].id
+        self.model_args["deposition_id"] = self.input_data["deposition"].id
```

Comment:
We currently don't have a metadata for this, so would we populating this from the dataset's deposition_id ?